src/cpp/session/modules/SessionHelp.cpp

```diff
@@ -732,6 +732,65 @@ r_util::RPackageInfo packageInfoForRd(const FilePath& rdFilePath)
    }
 }
 
+Error Rd2HTML(FilePath filePath, std::string* html)
+{
+   r::exec::RFunction fun(".rs.Rd2HTML", filePath.getAbsolutePath());
```

Comment:
You may want to use the `addUtf8Param()` helper here, e.g.

```suggestion
   r::exec::RFunction fun(".rs.Rd2HTML");
   fun.addUtf8Param(filePath.getAbsolutePath());
```

Otherwise, this might not work as expected for older versions of R on Windows, if the path cannot be represented in the native encoding.